src/views/community/components/community-post-menu.tsx

```diff
@@ -56,11 +55,9 @@ export default function CommunityPostMenu({
         <MenuItem onClick={() => window.navigator.clipboard.writeText("nostr:" + address)} icon={<RepostIcon />}>
           Copy Share Link
         </MenuItem>
-        {noteId && (
-          <MenuItem onClick={() => window.navigator.clipboard.writeText(noteId)} icon={<CopyToClipboardIcon />}>
-            Copy Note ID
-          </MenuItem>
-        )}
+        <MenuItem onClick={() => window.navigator.clipboard.writeText(event.id)} icon={<CopyToClipboardIcon />}>
```

Comment:
Probably should keep this the note1... most users don't have a use for the raw hex id of the note